apisix/init.lua

```diff
@@ -722,7 +722,19 @@ function _M.http_access_phase()
         plugin.run_plugin("access", plugins, api_ctx)
     end
 
-    _M.handle_upstream(api_ctx, route, enable_websocket)
+    if not api_ctx.custom_upstream_ip then
+        _M.handle_upstream(api_ctx, route, enable_websocket)
+    end
```

Comment:
There is no longer any place to use `ctx.custom_upstream_ip`, this should be modified.

Reply:
done